packages/@glimmer/runtime/test/in-element-test.ts

```diff
@@ -1,487 +1,441 @@
+
+    this.rerender({ bar: "World!" });
+    equalsElement(firstElement, "div", {}, stripTight`[GoodBye!]<!---->`);
+    equalsElement(secondElement, "div", {}, stripTight`[World!]`);
+    this.assertHTML("<!---->");
+    this.assertStableRerender();
+
+    this.rerender({ foo: "Hello!" });
+    equalsElement(firstElement, "div", {}, stripTight`[Hello!]<!---->`);
+    equalsElement(secondElement, "div", {}, stripTight`[World!]`);
+    this.assertHTML("<!---->");
+    this.assertStableRerender();
+  }
+
+  @test "Components are destroyed"() {
+    let destroyed = 0;
+    let DestroyMeComponent = EmberishCurlyComponent.extend({
+      destroy(this: EmberishCurlyComponent) {
+        this._super();
+        destroyed++;
+      }
+    });
+    this.env.registerEmberishGlimmerComponent("destroy-me", DestroyMeComponent as any, "destroy me!");
+    let externalElement = document.createElement("div");
+
+    this.render(
+      stripTight`
+        {{#if showExternal}}
+          {{#-in-element externalElement}}[{{destroy-me}}]{{/-in-element}}
+        {{/if}}
       `,
-    {
-      firstElement,
-      secondElement,
-      foo: 'Hello!',
-      bar: 'World!'
-    }
-  );
-
-  equalsElement(firstElement, 'div', {}, stripTight`[Hello!]<!---->`);
-  equalsElement(secondElement, 'div', {}, stripTight`[World!]`);
-
-  set(view, 'foo', 'GoodBye!');
-  rerender();
-
-  equalsElement(firstElement, 'div', {}, stripTight`[GoodBye!]<!---->`);
-  equalsElement(secondElement, 'div', {}, stripTight`[World!]`);
-
-  set(view, 'bar', 'Folks!');
-  rerender();
-
-  equalsElement(firstElement, 'div', {}, stripTight`[GoodBye!]<!---->`);
-  equalsElement(secondElement, 'div', {}, stripTight`[Folks!]`);
-
-  set(view, 'bar', 'World!');
-  rerender();
-
-  equalsElement(firstElement, 'div', {}, stripTight`[GoodBye!]<!---->`);
-  equalsElement(secondElement, 'div', {}, stripTight`[World!]`);
-
-  set(view, 'foo', 'Hello!');
-  rerender();
-
-  equalsElement(firstElement, 'div', {}, stripTight`[Hello!]<!---->`);
-  equalsElement(secondElement, 'div', {}, stripTight`[World!]`);
-});
-
-QUnit.test('components are destroyed', function(assert) {
-  let destroyed = 0;
-  let DestroyMeComponent = EmberishCurlyComponent.extend({
-    destroy(this: EmberishCurlyComponent) {
-      this._super();
-      destroyed++;
-    }
-  });
-
-  env.registerEmberishCurlyComponent('destroy-me', DestroyMeComponent as any, 'destroy me!');
-
-  let externalElement = document.createElement('div');
-
-  appendViewFor(
-    stripTight`
-      {{#if showExternal}}
-        {{#-in-element externalElement}}[{{destroy-me}}]{{/-in-element}}
-      {{/if}}
-    `,
-    {
-      externalElement,
-      showExternal: false,
-    }
-  );
-
-  equalsElement(externalElement, 'div', {}, stripTight``);
-  assert.equal(destroyed, 0, 'component was destroyed');
-
-  set(view, 'showExternal', true);
-  rerender();
-
-  assertElementIsEmberishElement(externalElement.firstElementChild!, 'div', { }, 'destroy me!');
```

Comment:
@chadhietala sorry that probably wasn't very clear. What I meant is that I couldn't use the `assertElementIsEmberishElement` method in the new tests and instead just did  `equalsElement`. My question was if this is an issue or if it's fine.